site/community.md

```diff
@@ -73,6 +73,26 @@ Here are some opportunities to meet with members of the community to learn and s
 
 <div class="row">
   <div class="col-md-6">
+    <p><img src="/assets/images/events/openexpo2017.png" alt="OpenExpo 2017" /></p>
+    <p><a href="http://www.openexpo.es/en/">OpenExpo 2017</a> <br /> June 1, 2017 in Madrid, Spain</p>
+  </div>
+</div>
+
+### Past Events
+
+<div class="row">
+  <div class="col-md-6">
+    <p><img src="/assets/images/events/rhsummit2017.png" alt="RHSUMMIT 2017" /></p>
+    <p><a href="https://www.redhat.com/en/summit/2017">Red Hat Summit 2017</a> <br /> May 2-4, 2017 in Boston, MA</p>
+  </div>
+  <div class="col-md-6">
+    <p><img src="/assets/images/events/OSCAL17.png" alt="OSCAL 2017" /></p>
```

Comment:
OSCAL17.png -> oscal17.png (case sensitive)

Rest LGTM. @Terones please make the change and I'll merge. Thanks!